kubernetes/hmda-spark-reporting/hmda-spark-aggregate.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     "spark.streaming.receiver.maxRate": "1"
     "spark.streaming.backpressure.initialRate": "1"
     "spark.streaming.backpressure.enabled": "true"
-  image: "omnipresent/aggregate-only9-2"
+  image: "omnipresent/fix-median-race"
```

Comment:
spoke with @Omnipresent  , this reference to the personal  docker repo will be removed in a future PR .